### ISSUE TITLE
Remove unnecessary condition

### ISSUE THREAD

**Issue by vweevers** [Level/subleveldown#82]
https://github.com/Level/subleveldown/blob/3e8c1af0bf911800666a347a4178ebf802aeb70f/leveldown.js#L89-L96

The `&& subdb.prefix` here should not be needed, because at this point we've already determined `subdb` to be a subleveldown instance and such an instance always has a prefix (even when the user passes in an empty prefix).

If it doesn't have a prefix, that's either a bug or incorrect usage (someone modifying the prefix). Either way it should be surfaced as an error, which is preferable to hitting the `else` branch on L94, which can result in an incorrect `self.leveldown`.

**Comment by vweevers** [comment]
No longer relevant after #89.
